app/assets/stylesheets/modules/homepage.scss

```diff
@@ -39,3 +39,83 @@ body.home-page {
     background-color: #008080;
   }
 }
+
+//SEARCH FORM
+.box{
+  width: 100%;
+  height: 50px;
+}
+
+.container-4{
+  overflow: hidden;
+  width: 100%;
+  vertical-align: middle;
+  white-space: nowrap;
+}
+
+.container-4 #location{
+  width: 100%;
+  height: 50px;
+  background: #ffffff;
+  border: none;
+  font-size: 10pt;
+  float: left;
+  color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.